lib/path/posix.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = require('path').posix;
```

Comment:
what's the benefit of the alias if it's requiring the full thing anyways?

Reply:
It makes it possible to import the **POSIX**‑specific path instance as a built‑in **ES Module** from **ES Modules**:
```js
import { join } from "path/posix";
```

Without this, that would have to be:
```js
import { posix } from "path";
const { join } = posix;
```

Whereas **CommonJS** modules can do:
```js
const { join } = require("path").posix;
```

---

It’s necessary to re‑export from `path`, as the **Win32** and **POSIX** path instances are created in the `path` module.

There’s also a circular dependency between `path.posix` and `path.win32`:
`path.win32.posix === path.posix` and `path.posix.win32 === path.win32`.